public/keymaps/percent_canoe_default.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Canoe actually has a default but it never got renamed when we added the `percent` folder on qmk_firmware.

Could you dump these contents into `public/keymaps/canoe_default.json` and rename that file to this filename instead?

Reply:
Done 👍